specification/Metadata/1.0.0/README.md

```diff
@@ -0,0 +1,688 @@
+<!-- omit in toc -->
+# Cesium 3D Metadata Specification
+
+**Version 1.0.0**, February 25, 2021
+
+<!-- omit in toc -->
+## Contributors
+
+* Peter Gagliardi, Cesium
+* Sean Lilley, Cesium
+* Sam Suhag, Cesium
+* Patrick Cozzi, Cesium
+* Bao Tran, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Concepts](#concepts)
+- [Schemas](#schemas)
+  - [Classes and Properties](#classes-and-properties)
+  - [Property Types](#property-types)
+  - [Arrays](#arrays)
+  - [Enums](#enums)
+  - [Normalized Properties](#normalized-properties)
+  - [Minimum and Maximum Values](#minimum-and-maximum-values)
+  - [Optional Properties and Default Values](#optional-properties-and-default-values)
+  - [Semantics](#semantics)
+- [Storage Formats](#storage-formats)
+  - [Table Format](#table-format)
+    - [Binary Encoding](#binary-encoding)
+      - [Numbers](#numbers)
+      - [Booleans](#booleans)
+      - [Strings](#strings)
+      - [Enums](#enums-1)
+      - [Arrays](#arrays-1)
+    - [JSON Encoding](#json-encoding)
+      - [Numbers](#numbers-1)
+      - [Booleans](#booleans-1)
+      - [Strings](#strings-1)
+      - [Enums](#enums-2)
+      - [Arrays](#arrays-2)
+  - [Raster Format](#raster-format)
+    - [Numbers](#numbers-2)
+    - [Enums](#enums-3)
+    - [Arrays](#arrays-3)
+    - [Implementation Notes](#implementation-notes)
+
+## Overview
+
+The Cesium 3D Metadata Specification defines a standard metadata format for 3D data. This spans a myriad of domains, such as heights of buildings in a city, names of different structures in a CAD model, material properties for textured surfaces, and classification codes for point clouds.
+
+This specification defines a set of core concepts to be used by multiple 3D formats. It is language and format independent. This document occasionally uses JSON examples for illustrating concepts but does not define a JSON schema.
+
+For usage see:
+
+* [`3DTILES_metadata`](../../../extensions/3DTILES_metadata/1.0.0) - 3D Tiles extension that assigns metadata to various components of 3D Tiles
```

Comment:
To be consistent with other wording of these specs, we think that it would be good to talk about "at different levels of granularity" and then give examples (tileset, tile, feature)

Reply:
Sounds good.